crates/ruff/src/importer/mod.rs

```diff
@@ -18,10 +18,16 @@ use crate::importer::insertion::Insertion;
 mod insertion;
 
 pub(crate) struct Importer<'a> {
+    /// The Python AST to which we are adding imports.
     python_ast: &'a Suite,
+    /// The [`Locator`] for the Python AST.
     locator: &'a Locator<'a>,
+    /// The [`Stylist`] for the Python AST.
     stylist: &'a Stylist<'a>,
-    ordered_imports: Vec<&'a Stmt>,
+    /// The list of visited, top-level runtime imports in the Python AST.
+    runtime_imports: Vec<&'a Stmt>,
+    /// The list of visited, top-level `if TYPE_BLOCKING:` blocks in the Python AST.
```

Comment:
Is this supposed to be TYPE_CHECKING?

```suggestion
    /// The list of visited, top-level `if TYPE_CHECKING:` blocks in the Python AST.
```